src/routes/authentication-route.js

```diff
@@ -12,7 +12,14 @@ module.exports = function createAuthenticationRoute(repository) {
     const result = await repository.getUser(request.body);
     if (!result) return response.sendStatus(404);
     const token = jwt.sign({ role: result[0], user: result[1].id }, config.jwtconf.secret, config.jwtconf.time);
-    return response.status(200).json({ token });
+    const userData = {
+      id: result[1].id,
```

Comment:
Why would you need to get the first element of the array. `getUser` should return the current user as object, if a function does 2 things then it is a violation of single responsibility.

Reply:
get users is an array have 2 elements, 
1- role
2 - user data
we need everything from date, except password
for not rewriting a new method, we just created a new object, that contains everything , except password